website/docs/packages/reference/package.md

```diff
@@ -0,0 +1,56 @@
+# Package Settings
+
+This page provides information about the settings and properties available within Packages, including configuration options for query modules and JavaScript modules.
+
+## Query module properties
```

Comment:
LEt's call settings only. Properties are referred to as the ones modified from the properties pane.